backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe "spree/admin/shared/_navigation_footer", type: :view do
+  let(:user) { create(:admin_user) }
```

Comment:
`Spree.user_class.new` can be used. Of course, as you see fit. Just in terms of speed, not persisting data where possible I think is preferable. 

And yeah .. making the start with the first `view` spec is indeed the toughest, so great on doing that! Might encourage others to follow suite as they are easy to write, and quick to run.